src/pyuvdata/telescopes.py

```diff
@@ -400,30 +402,79 @@ def __init__(self):
         )
 
         desc = (
-            "Orientation of the physical dipole corresponding to what is "
-            "labelled as the x polarization. Options are 'east' "
-            "(indicating east/west orientation) and 'north (indicating "
-            "north/south orientation)."
+            "Antenna diameters in meters. Used by CASA to "
+            "construct a default beam if no beam is supplied."
         )
-        self._x_orientation = uvp.UVParameter(
-            "x_orientation",
+        self._antenna_diameters = uvp.UVParameter(
+            "antenna_diameters",
             description=desc,
             required=False,
+            form=("Nants",),
+            expected_type=float,
+            tols=1e-3,  # 1 mm
+        )
+
+        desc = (
+            'Antenna mount type, shape (Nants,). Options are: "alt-az", "equitorial", '
+            '"orbiting", "x-y", "alt-az+nasmyth-r", "alt-az+nasmyth-l", "phased", '
+            '"fixed", and "other".'
```

Comment:
If we're not going to say more than this I think we need to point to the documentation you're planning to add on what these mean. I find these names confusing and even misleading (especially "x-y") so it's important to tell users where to get more info..

Reply:
Agreed - I'm gonna expound on this a bit in the docstring and point toward the conventions page for further details (writing that now)...

Reply:
Okay, added in 11d373ecdf270b073d714514c43f9a1bd55e21d0.